concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -753,3 +786,25 @@ def test_download_should_return_401_if_file_categories_are_not_unique_across_fil
         self.assertIn('error_code', response.json().keys())
         self.assertEqual("application/json", response["Content-Type"])
         self.assertEqual(response.json()["error_code"], ErrorCode.MESSAGE_FILES_CATEGORY_NOT_UNIQUE.value)
+
+    @freeze_time("2018-12-30 11:00:00")
+    def test_upload_should_return_401_if_invalid_protocol_version(self):
```

Comment:
wrong test name "upload" -> "download".
besides, is `GOLEM_MESSAGES_VERSION` relevant for file transfer operations?

Reply:
renamed. At second question @rwrzesien answered above